modular_skyrat/modules/modular_implants/code/nifsofts/money_sense.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/disk/nifsoft_uploader/money_sense
+	name = "Automatic Apprasial"
+	loaded_nifsoft = /datum/nifsoft/money_sense
+
+/datum/nifsoft/money_sense
+	name = "Automatic Appraisal" //Placeholder name until I figure out something cooler
+	program_desc = "Connects the user's brain to a database containing the current monetary values for most items, allowing them to determine their value in realtime"
+	active_mode = TRUE
+	active_cost = 0.5
+	compatible_nifs = list(/obj/item/organ/internal/cyberimp/brain/nif/standard)
+
+/datum/nifsoft/money_sense/activate()
+	. = ..()
+	if(active)
+		linked_mob.money_sense = TRUE
+		return
+
+	if(linked_mob.money_sense)
+		linked_mob.money_sense = FALSE
+
+/mob/living/carbon
+	///Is the mob able to see the monetary value of items by examining them?
+	var/money_sense = FALSE
+
+/obj/item/examine(mob/user)
+	. = ..()
+
+	var/mob/living/carbon/looking_mob = user
+
+	if(looking_mob) //This nesting isn't ideal, but early returning may cause issues.
+		if(looking_mob.money_sense)
```

Comment:
this runtimes whenever observers try to examine items, further reason to just componentize these

Reply:
will do